test/errorHandling.js

```diff
@@ -336,6 +340,17 @@ describe('Error Handling Tests:', function() {
       expect(result).to.deep.equal({ multiValueHeaders: { 'content-type': ['application/json'] }, statusCode: 500, body: '{"error":"This is a test error message"}', isBase64Encoded: false })
     }) // end it
 
+    it('Should not log error if option logger.errorLogging is false', async function() {
```

Comment:
Perhaps add a test for `errorLogging: true`?

Reply:
Good idea!